src/zrepl_mgmt.c

```diff
@@ -78,7 +78,11 @@ zrepl_log(enum zrepl_log_level lvl, const char *fmt, ...)
 	va_start(args, fmt);
 	vsnprintf(line + off, sizeof (line) - off, fmt, args);
 	va_end(args);
-	fprintf(stderr, "%s\n", line);
+	if (lvl == LOG_LEVEL_ERR) {
```

Comment:
this 'if-else' doesn't look good

Reply:
Can we have separate func `zrepl_err_log`? Or not touching this code and at cstor side can we log import logs as error instead of info and mgmt will print by reading from stderr.

Reply:
Or do you have any suggestions regarding this? 
cc: @vishnuitta